src/AMAMM.sol

```diff
@@ -180,13 +185,13 @@ contract AMAMM is IAmAmm {
     function _updateLastUpdatedEpoch(PoolId id, uint40 _epoch) internal returns (uint40) {
         if (_lastUpdatedEpoch[id] > _getEpoch(id, block.timestamp)) {
```

Comment:
It could be simplified by just comparing `_epoch < _lastUpdatedEpoch[id]`, no need to compare it with `_getEpoch`.